lib/widget/dropdown.dart

```diff
@@ -132,6 +126,39 @@ abstract class SelectOne extends StatefulWidget with Invokable, HasController<Se
 
   }
 
+  bool isIconExist(List v)
+  {
+    bool isIcon = false;
+    for (var e in v) {
+      if (e is Map) {
+        if (e['icon'] != null) {
+          isIcon = true;
+        }
+      }
+    }
+    return isIcon;
+  }
+
+  void calculateIconSize(List v) {
+    List iconSize = [];
+    _controller.size = 0;
+    for (var e in v) {
+      if (e is Map) {
+        if (e['icon'] != null) {
+          if (e['icon']['size'] != null) {
+            iconSize.add(e['icon']['size']);
```

Comment:
nothing big here but why not just do Math.max(currentSize, newSize) so you don't mess with adding/removing array items?

Reply:
basically 1st is check icon exist or not in all array than is exist than is check how many time icon size are exist in array that 's why i added all icon sizes in are and find max value